gradle.properties

```diff
@@ -2,3 +2,4 @@ version=0.12.0-SNAPSHOT
 kestraVersion=[0.12,)
 micronautVersion=3.9.3
 lombokVersion=1.18.28
+surrealdbVersion=0.1.0
```

Comment:
Not used.
When there is only one lib that uses this version, adding it here is not needed as the version can be specified directly in the build.gradle as you did.